tests/test_discounts.py

```diff
@@ -252,10 +252,12 @@ def test_sale_applies_to_correct_products(product_class):
 
 
 @pytest.mark.django_db
-def test_get_category_variants_and_prices_product_with_many_categories(cart, default_category, product_in_stock):
-    """Test error: get_category_variants_and_prices return duplicated items. It cause double(or more) percentage discount"""
+def test_get_category_variants_and_prices_product_with_many_categories(cart, default_category,
+                                                                       product_in_stock):
+    # Test error: get_category_variants_and_prices return duplicated items.
```

Comment:
We don't need to test errors. Describe what this function tests and what is expected output.